extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -438,6 +440,9 @@ export class ConsentUI {
     toggle(dev().assertElement(this.ui_), false);
 
     const iframePromise = this.getClientInfoPromise_().then(clientInfo => {
+      if (opt_promptTrigger) {
+        clientInfo['promptTrigger'] = opt_promptTrigger;
```

Comment:
`opt_promptTrigger` should be a boolean internally,

the line would be come:

clientInfo['promptTrigger'] = opt_promptTrigger ? 'action' : 'load';

`action` - Caused by an action, like a button click.

`load` - Prompt was triggered on page load (force prompt, unknown consent state) 

This is in response to @zhouyx 's comment. Thanks for helping in the bikeshedding @calebcordry 😄